### DISCUSSION TITLE
Isolated field-specific validation?

### DISCUSSION THREAD

**Discussion by tjdam** [react-hook-form/react-hook-form#7540] [Q&A] [answered]
Hi!

I have this field in my form that is validated using Yup and one of the fields is validated against a remote API.
The validation mode is set to 'onChange', which is pivotal to help the user know as they type if they got the correct value.

The problem is, this validation is called for every single change in the form, even when the field in question is not being altered.

I was wondering if there is a way to isolate validation to this one specific field.

Thank you for the great work, btw!

Oh, I don't know how much different things are now, but I'm currently stuck on v6....

**Comment by bluebill1049** [comment, marked as answer]
you can use `trigger` and customise when to fire the validation.

**Reply by tjdam** [reply]
Ohhh!

I thought `trigger` would trigger additional validation, I didn't realize it separates the field validations from the rest!

Thank you!
